workflows/utilities/wf_organism_parameters.wdl

```diff
@@ -49,6 +60,19 @@ workflow organism_parameters {
     Int sc2_vadr_skip_length = 10000
     String sc2_vadr_options = "--noseqnamemax --glsearch -s -r --nomisc --mkey sarscov2 --lowsim5seq 6 --lowsim3seq 6 --alt_fail lowscore,insertnn,deletinn --out_allfasta"
     Int sc2_vadr_memory = 8
+
+    # augur options for sars-cov-2
+    #File sc2_lat_longs_tsv = "gs://theiagen-public-files-rp/terra/flu-references/lat_longs.tsv"
+    #File sc2_clades_tsv = "gs://theiagen-public-files-rp/terra/sars-cov-2-files/sc2_clades.tsv"
+    #File sc2_reference_gbk = "gs://theiagen-public-files-rp/terra/sars-cov-2-files/MN908947.3.gb"
+    #File sc2_auspice_config = "gs://theiagen-public-files-rp/terra/sars-cov-2-files/sc2_auspice_config.json"
+    #Int sc2_min_num_unambig = 27000
+    #Float min_date = 2020.0
+    #Int pivot_interval = 1
+    #String pivot_interval_units = "weeks"
+    #Float narrow_bandwidth = 0.05
+    #Float proportion_wide = 0.0
```

Comment:
Do we intend to keep these commented out lines 64-74? Will they be useful in the future?

I saw the note about SC2 nextstrain files being frequently updated (like the clades.tsv file) so I wasn't sure if we should delete these lines or not.

If we don't intend to keep them for future use, could you please delete these lines?

Reply:
Only if we plan to use augur_frequencies, task present but currently not used. Removing for SARS-CoV-2 till when needed for a cleaner look. These were set for the other pathogens, but also not used presently.